cmake/Coverage.cmake

```diff
@@ -33,6 +33,12 @@ if(NRN_ENABLE_COVERAGE)
   set(NRN_COVERAGE_FLAGS_UNQUOTED --coverage -O0 -fno-inline -g)
   set(NRN_COVERAGE_FLAGS "--coverage -O0 -fno-inline -g")
   set(NRN_COVERAGE_LIB gcov)
+
+  if (NRN_MACOS_BUILD)
+    unset(NRN_COVERAGE_LIB)
+    add_link_options(-fprofile-arcs)
+  endif()
```

Comment:
if these options are compiler specific (gnu vs clang), do we need extra guard around them? Or, info about which compiler can be used for code coverage?

Reply:
That is a good point. This PR solved my problem on the Mac M1 but I don't know under what circumstances, if any,  this PR is inappropriate.